piqueserver/server.py

```diff
@@ -272,6 +272,8 @@ def __init__(self, interface: bytes, config_dict: Dict[str, Any]) -> None:
         self.win_count = itertools.count(1)
         self.bans = NetworkDict()
 
+        self.proto_extensions = [(EXTENSION_CHATTYPE, 1)]
```

Comment:
I would make it clearer that this is the list of extensions the server *theoretically* supports. With it being the same variable name as the client where it is the things the client *actually* supports, it could get confusing.

Reply:
Any name suggestion? I though about just changing it to "extensions", but i think it would lead to confusion with map extensions, or even with the extensions.py file.

I kept as proto_extensions, because it reminds me of protocol_extensions.

Reply:
how about `available_proto_extensions`? My main worry is people accidentally checking if the *server* supports supports something instead of the client. If they have the same name that's very easy.

Reply:
That would work, i guess